app/views/claims/correct_school.html.erb

```diff
@@ -7,7 +7,7 @@
 
     <legend class="govuk-fieldset__legend govuk-fieldset__legend--<%= shared_view_css_size %>">
       <h1 class="govuk-fieldset__heading">
-        <%= t("additional_payments.questions.current_school_search") %>
+        <%= t("additional_payments.forms.current_school.questions.current_school_search") %>
```

Comment:
We are using a journey-specific translation in a non-journey-specific generic template. We probably need to have a I18n namespace for non-specific translations, or we might need to duplicate or move this template to the journey folder if it needs to be specific.

Reply:
OK, in that case this template should live in the `views/additional_payments/claims` folder and not in the `views/claims` folder.